### PR TITLE
test(composer-app): e2e test fixes

### PR DESCRIPTION
**Pull request by wittjosiah**

### CODE REVIEW

**Review by thure — APPROVED** [review]
Thank you ✨

File: packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx [hunk 1]
```diff
@@ -94,7 +94,7 @@ export const DeckLayout = ({
     }
   }, [layoutMode, activeId]);
 
-  const parts: LayoutEntry[] = layoutParts.main ?? layoutParts.solo ?? [];
+  const parts: LayoutEntry[] = (layoutMode === 'solo' ? layoutParts.solo : layoutParts.main) ?? [];
```
Comment (ZaymonFC) [inline packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:97]: This is superseded by https://github.com/dxos/dxos/pull/7647/files#diff-0c185a2f7717ff0db23f06575e664180f4c8b2fd8c217bb373d61a74b78c8eeeR96-R109?

Comment (wittjosiah) [reply packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:97]: yeah, it is. once that lands I'll rebase this and merge it as just e2e test fixes

Comment (ZaymonFC) [reply packages/plugins/plugin-deck/src/components/DeckLayout/DeckLayout.tsx:97]: 👍

**Pull request merged**